File: Java_Virtual_World/src/Gameplay/Animals/Sheep.java
package Gameplay.Animals;

import Gameplay.Organism;

public class Sheep extends Animal {
    public static final int STRENGTH = 4;
    public static final int INITIATIVE = 4;

    public Sheep(int x, int y) {
        super(x, y, STRENGTH, INITIATIVE);
    }
    @Override
    public String Draw(){
        return "🐑";
    }
    @Override
    public Organism Copy(){
        return new Sheep(x,y);
    }
    @Override
    public String toString(){
        return "Sheep";
    }

}

File: Python_Virtual_World/Sheep.py
from MyVector import MyVector
from Animal import Animal


class Sheep(Animal):


    STRENGTH = 4
    INITIATIVE = 4


    def __init__(self, position : MyVector):
        super().__init__(position, Sheep.STRENGTH, Sheep.INITIATIVE)


    def draw(self) -> str:

        return "gray"


    def __str__(self):
        return "Sheep"
